packages/discord.js/typings/index.d.ts

```diff
@@ -1562,6 +1562,11 @@ export class InteractionWebhook extends PartialWebhookMixin() {
   public constructor(client: Client, id: Snowflake, token: string);
   public token: string;
   public send(options: string | MessagePayload | InteractionReplyOptions): Promise<Message>;
+  public editMessage(
+    message: MessageResolvable,
+    options: string | MessagePayload | WebhookEditMessageOptions,
+  ): Promise<Message>;
```

Comment:
This is technically not gonna be true if you use no intents in your bot (which is possible)...

Reply:
But it won't have the channel in cache 👀